Fails to build on darwin:

```
building
build flags: -j4 -l4 SHELL=/nix/store/k89nm2jva0qmvd970f84wq2iq1iwm9bs-bash-4.4-p23/bin/bash
Scanning dependencies of target mympd
Scanning dependencies of target mympd-script
Scanning dependencies of target mympd-config
[  1%] Building C object cli_tools/CMakeFiles/mympd-script.dir/__/dist/src/mongoose/mongoose.c.o
[  2%] Building C object cli_tools/CMakeFiles/mympd-config.dir/mympd-config.c.o
[  3%] Building C object cli_tools/CMakeFiles/mympd-script.dir/mympd-script.c.o
[  3%] Building C object CMakeFiles/mympd.dir/src/main.c.o
clang-7: error: argument unused during compilation: '-pie' [-Werror,-Wunused-command-line-argument]
clang-7: error: argument unused during compilation: '-s' [-Werror,-Wunused-command-line-argument]
make[2]: *** [cli_tools/CMakeFiles/mympd-script.dir/build.make:95: cli_tools/CMakeFiles/mympd-script.dir/__/dist/src/mongoose/mongoose.c.o] Error 1
make[2]: *** Waiting for unfinished jobs....
clang-7: error: argument unused during compilation: '-pie' [-Werror,-Wunused-command-line-argument]
clang-7: error: argument unused during compilation: '-s' [-Werror,-Wunused-command-line-argument]
clang-7: error: argument unused during compilation: '-pie' [-Werror,-Wunused-command-line-argument]
clang-7: error: argument unused during compilation: '-s' [-Werror,-Wunused-command-line-argument]
make[2]: *** [cli_tools/CMakeFiles/mympd-config.dir/build.make:82: cli_tools/CMakeFiles/mympd-config.dir/mympd-config.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:171: cli_tools/CMakeFiles/mympd-config.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
make[2]: *** [cli_tools/CMakeFiles/mympd-script.dir/build.make:82: cli_tools/CMakeFiles/mympd-script.dir/mympd-script.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:144: cli_tools/CMakeFiles/mympd-script.dir/all] Error 2
clang-7: error: argument unused during compilation: '-pie' [-Werror,-Wunused-command-line-argument]
clang-7: error: argument unused during compilation: '-s' [-Werror,-Wunused-command-line-argument]
make[2]: *** [CMakeFiles/mympd.dir/build.make:82: CMakeFiles/mympd.dir/src/main.c.o] Error 1
make[2]: *** Waiting for unfinished jobs....
[  4%] Building C object CMakeFiles/mympd.dir/src/api.c.o
[  5%] Building C object CMakeFiles/mympd.dir/src/list.c.o
[  6%] Building C object CMakeFiles/mympd.dir/src/global.c.o
clang-7: error: argument unused during compilation: '-pie' [-Werror,-Wunused-command-line-argument]
clang-7: error: argument unused during compilation: '-s' [-Werror,-Wunused-command-line-argument]
make[2]: *** [CMakeFiles/mympd.dir/build.make:95: CMakeFiles/mympd.dir/src/api.c.o] Error 1
clang-7: error: argument unused during compilation: '-pie' [-Werror,-Wunused-command-line-argument]
clang-7: error: argument unused during compilation: '-s' [-Werror,-Wunused-command-line-argument]
clang-7: error: argument unused during compilation: '-pie' [-Werror,-Wunused-command-line-argument]
clang-7: error: argument unused during compilation: '-s' [-Werror,-Wunused-command-line-argument]
make[2]: *** [CMakeFiles/mympd.dir/build.make:108: CMakeFiles/mympd.dir/src/global.c.o] Error 1
make[2]: *** [CMakeFiles/mympd.dir/build.make:121: CMakeFiles/mympd.dir/src/list.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:117: CMakeFiles/mympd.dir/all] Error 2
make: *** [Makefile:149: all] Error 2
```

--- a/pkgs/applications/audio/mympd/default.nix
+++ b/pkgs/applications/audio/mympd/default.nix
@@ -1,0 +1,53 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, mpd_clientlib
+, openssl
+, lua5_3
+, libid3tag
+, flac
+, mongoose
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mympd";
+  version = "6.8.1";
+
+  src = fetchFromGitHub {
+    owner = "jcorporation";
+    repo = "myMPD";
+    rev = "v${version}";
+    sha256 = "dIGg2mLxN6XBDH3GFXtF7nB9a/zf/qMlPCvIulFRXn8=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    cmake
+  ];
+  buildInputs = [
+    mpd_clientlib
+    openssl
+    lua5_3
+    libid3tag
+    flac
+  ];
+
+  cmakeFlags = [
+    "-DENABLE_LUA=ON"
+    # Otherwise, it tries to parse $out/etc/mympd.conf on startup.
+    "-DCMAKE_INSTALL_SYSCONFDIR=/etc"
+    # similarly here
+    "-DCMAKE_INSTALL_LOCALSTATEDIR=/var/lib/mympd"
+  ];
+  # See https://github.com/jcorporation/myMPD/issues/315
+  hardeningDisable = [ "strictoverflow" ];
+
+  meta = {
+    homepage = "https://jcorporation.github.io/mympd";
+    description = "A standalone and mobile friendly web mpd client with a tiny footprint and advanced features";
+    maintainers = [ stdenv.lib.maintainers.doronbehar ];
+    platforms = stdenv.lib.platforms.linux;
+    license = stdenv.lib.licenses.gpl2Plus;
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -22556,6 +22556,9 @@ in
 
   ympd = callPackage ../applications/audio/ympd { };
 
+  # a somewhat more maintained fork of ympd
+  mympd = callPackage ../applications/audio/mympd { };
+
   nload = callPackage ../applications/networking/nload { };
 
   normalize = callPackage ../applications/audio/normalize { };

